src/BookingForm.js

```diff
@@ -0,0 +1,195 @@
+import React, { useState } from "react";
+import SearchResults from "./SearchResults.js";
+import TableRow from "./TableRow.js";
+function BookingForm({ handleNewBooking }) {
+  const [isSubmitClicked, setSubmitClicked] = useState(false);
+  const [newBookingInfo, setNewbookingInfo] = useState({
+    id: "0",
+    title: "",
+    firstName: "",
+    surname: "",
+    email: "",
+    roomId: "",
+    checkInDate: "",
+    checkOutDate: "",
+    items: []
+  });
+
+  function handleSelect(event) {
+    let newId = 5;
+    newId++;
+    const updatedCustomerInfo = {
+      id: newId,
+      title: event.target.value
+    };
+    setNewbookingInfo(updatedCustomerInfo);
+  }
+
+  function handleChange(event) {
+    const updatedCustomerInfo = {
+      ...newBookingInfo,
+      [event.target.name]: event.target.value
+    };
+    setNewbookingInfo(updatedCustomerInfo);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    let items = [];
```

Comment:
Not sure why you need this `items` array - you can update the array of bookings at the parent level. The only thing you will need to be concerned with when the submit event fires is passing the new booking from the form up to the parent component ( `App.js`, I believe )

Reply:
DM me if you need some help fixing this one